_posts/en/podcast/2023-09-04-waiting-for-confirmation-special.md

```diff
@@ -0,0 +1,30 @@
+---
+title: 'Bitcoin Optech Podcast Special: Waiting for confirmation'
+permalink: /en/podcast/2023/09/waiting-for-confirmation/
+reference: /en/blog/waiting-for-confirmation/
+name: 2023-09-04-waiting-for-confirmation
+slug: 2023-09-04-waiting-for-confirmation
+type: podcast
+layout: podcast-episode
+lang: en
+---
+This podcast is a compilation of previous podcast discussions about the [Waiting
+for confirmation]({{page.reference}}) series on mempool and relay policy. Series
+coauthors Mark "Murch" Erhardt and Gloria Zhao describe the concepts in the
```

Comment:
I would like to suggest mentioning Gloria first, both alphabetically and by contribution. :)

```suggestion
coauthors Gloria Zhao and Mark "Murch" Erhardt describe the concepts in the
```